dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/parts/WebpageController.java

```diff
@@ -127,22 +141,17 @@
           Direction sortDirection,
       @RequestParam(name = "nullHandling", required = false, defaultValue = "NATIVE")
           NullHandling nullHandling,
-      @RequestParam(name = "publicationStart", required = false)
-          FilterCriterion<LocalDate> publicationStart,
-      @RequestParam(name = "publicationEnd", required = false)
-          FilterCriterion<LocalDate> publicationEnd)
+      @RequestParam(name = "active", required = false) String active)
       throws IdentifiableServiceException {
     Sorting sorting = null;
-    if (sortField != null) {
+    if (!Strings.isNullOrEmpty(sortField)) {
```

Comment:
StringUtils.
public static boolean isEmpty(@Nullable Object str) {
in spring utils

public static boolean isEmpty(@Nullable Object str) {
		return (str == null || "".equals(str));
	}

Reply:
Danke für den Hinweis, hab es getestet und ersetzt.